### PR TITLE
[DOC] Add alternative shell command to delete knative service 

### PR DESCRIPTION
**Pull request by AntoineToussaint**
## Proposed Changes

- Add alternative shell command to delete a `kservice`.

### CODE REVIEW

**Comment by AntoineToussaint** [comment]
assign @richieescarez

**Comment by AntoineToussaint** [comment]
@googlebot I signed it!

File: docs/serving/getting-started-knative-app.md [hunk 1]
```diff
@@ -183,3 +183,9 @@ To remove the sample app from your cluster, delete the service record:
 ```shell
 kubectl delete --filename service.yaml
 ```
+
+Alternatively, delete the service record by name:
```
Comment (sebgoa) [inline docs/serving/getting-started-knative-app.md]: I would remove "record"

Comment (AntoineToussaint) [reply docs/serving/getting-started-knative-app.md]: I removed the word "record", is that correct?

**Pull request merged**